docs/source/en/using-diffusers/loading.mdx

```diff
@@ -178,105 +198,321 @@ stable_diffusion_img2img = StableDiffusionImg2ImgPipeline(**components)
 
 Note how the above code snippet makes use of [`DiffusionPipeline.components`].
 
+### Loading variants
+
+Diffusion Pipeline checkpoints can offer variants of the "main" diffusion pipeline checkpoint.
+Such checkpoint variants are usually variations of the checkpoint that have advantages for specific use-cases and that are so similar to the "main" checkpoint that they **should not** be put in a new checkpoint. A variation of a checkpoint has to have **exactly** the same checkpoint layout including all weights having the same tensor shapes.
```

Comment:
A variation of a checkpoint has to have **exactly** the same serialization format and **exactly** the same checkpoint layout, including all weights having the same tensor shapes.

Reply:
BTW @yiyixuxu feel free to leave suggestions to correct directly in line: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request